src/koans/25_threading_macros.clj

```diff
@@ -0,0 +1,70 @@
+(ns koans.25-threading-macros
+  (:require [koan-engine.core :refer :all]))
+
+(def a-list
+  '(1 2 3 4 5))
+
+(def a-list-with-maps
+  '({:a 1} {:a 2} {:a 3}))
+
+(defn function-that-takes-a-map [m a b]
+  (do
+    (println (str "Other unused arguments: " a " " b))
```

Comment:
Can we eliminate the printing to STDOUT here? This causes extra output in `lein koan test` and `lein koan run` that I'd prefer to avoid - seems like they could be potentially confusing for learners too. Maybe you could use `str` the additional arguments together and return a vector of results or something?